nixos/hosts/vps/rathole.vm.toml

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: Andrew Hayzen <ahayzen@gmail.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+[server]
+bind_addr = "0.0.0.0:8333"
+default_token = "CHANGEME"
```

Comment:
link to the bug about multiple configs, which could then mean agenix only contains the token part?